ding/utils/data/collate_fn.py

```diff
@@ -63,6 +64,12 @@ def default_collate(batch: Sequence,
             # return torch.stack(batch, 0, out=out)
         else:
             return torch.stack(batch, 0, out=out)
+    elif isinstance(elem, ttorch.Tensor):
+        ret = ttorch.stack(batch).json()
+        for k in ret:
```

Comment:
Is it possible to convert to matrix operations here, and ret losses the data type here

Reply:
it has already been transformed into dict from treetensor, so just modify it in a for loop.